### DISCUSSION TITLE
Add search engine probably using server request

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [hoothin/SearchJumper#178] [Q&A] [answered]
Hi,

I don’t know if it’s possible because I don’t know how this site: https://ok.ru/video makes its request.

If you manage to add this search engine, let me know, please.

**Comment by hoothin** [comment, marked as answer]
`https://ok.ru/video#p{wait()&video-search-input input=%s&click([aria-label\=Search])}`

**Reply by Fred-Vatin** [reply]
Thanks but it works only if the search is ran in a tab where the `https://ok.ru/video` is already open.

So, I added delay to make it work : 
```
https://ok.ru/video#p{wait()&sleep(800)&video-search-input input=%s&sleep(400)&click([aria-label\=Search])}
```
Those sleep values work for me but I’m not sure it would work for everyone.

A better approach would be to wait a specific file is loaded (I don’t know if it is possible) or the required element in the DOM (but I didn’t find any for now).
